### PR TITLE
Feature/aqi 3742

### PR DESCRIPTION
**Pull request by KentMiller**

### CODE REVIEW

**Comment by joezhou-aqi** [comment]
Are there any unit tests for this body of work? @KentMiller

**Comment by KentMiller** [comment]
> 

@joezhou-aqi  - there are no unit tests in this solution. I created automated tests in the test fixture. There's a crucible review for this.

**Review by AQI-DanG — APPROVED** [review]

File: src/Aquarius.ONE.ClientSDK/Aquarius.ONE.ClientSDK.csproj [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	  <!--For Release packages update the VersionPrefix and remove any VersionSuffix-->
 	  <VersionPrefix>13.13.1</VersionPrefix>
 	  <!--For Feature packages update the VersionSuffix with the Jira ticket number-->
-	  <VersionSuffix></VersionSuffix>
+	  <VersionSuffix>aqi-3742-04</VersionSuffix>
```
Comment (JonathanRobertson-AQI) [inline src/Aquarius.ONE.ClientSDK/Aquarius.ONE.ClientSDK.csproj:8]: I'd vote we just push a new version (not prerelease)

File: src/Aquarius.ONE.ClientSDK/Aquarius.ONE.ClientSDK.csproj [hunk 2]
```diff
@@ -5,7 +5,7 @@
 	  <!--For Release packages update the VersionPrefix and remove any VersionSuffix-->
 	  <VersionPrefix>13.13.1</VersionPrefix>
```
Comment (JonathanRobertson-AQI) [inline src/Aquarius.ONE.ClientSDK/Aquarius.ONE.ClientSDK.csproj:6]: Please bump the version

**Review by JonathanRobertson-AQI — CHANGES_REQUESTED** [review]
I think the code changes make sense, but I'd suggest just creating a new package for your work and not doing a pre-release. @DarshanaGandhi-AI can create a new package when her changes are ready. Thanks!

File: src/Aquarius.ONE.ClientSDK/Aquarius.ONE.ClientSDK.csproj (hunk 1 above)
Comment (KentMiller) [reply src/Aquarius.ONE.ClientSDK/Aquarius.ONE.ClientSDK.csproj]: I will release version 13.14.0

File: src/Aquarius.ONE.ClientSDK/Aquarius.ONE.ClientSDK.csproj (hunk 2 above)
Comment (KentMiller) [reply src/Aquarius.ONE.ClientSDK/Aquarius.ONE.ClientSDK.csproj]: Bumped to version 13.14.0

**Comment by KentMiller** [comment]
>

**Review by JonathanRobertson-AQI — APPROVED** [review]

**Comment by JonathanRobertson-AQI** [comment]
I see that the package was created (https://www.nuget.org/packages/Aquarius.ONE.ClientSDK/13.14.0), which I think makes sense. I'd like to merge this to keep Master in sync with our package creation. Any push back?

**Pull request merged**